Formula/mariadb@10.5.rb

```diff
@@ -38,8 +38,8 @@ class MariadbAT105 < Formula
     # Need patch to remove MYSQL_SOURCE_DIR from include path because it contains
     # file called VERSION.
     # https://github.com/Homebrew/homebrew-core/pull/76887#issuecomment-840851149
-    # Reported upstream at https://jira.mariadb.org/browse/MDEV-7209 - this fix can be
-    # removed once that issue is closed and the fix has been merged into a stable release.
+    # Originally reported upstream at https://jira.mariadb.org/browse/MDEV-7209,
+    # but only partially fixed.
     patch :DATA
```

Comment:
Can we upstream this patch? It seems like upstream haven't fixed this particular issue yet.

Reply:
Yeah I think they incorrectly marked it as resolved when it is not.  I'm not sure whether the fix we use would actually be acceptable for them, but it's worth submitting.  Fortunately the fix is extremely minor at this point.

Reply:
I opened a PR to fix the `VERSION` issue for `mroonga` here: https://github.com/MariaDB/server/pull/1891.  Is it sufficient for to me to add this link as a comment for all versions of MariaDB?  If so then the MariaDB PRs should be ready to go.

Reply:
Yup, that should be fine, I think.

Reply:
Great, I'll follow up with that in a syntax only PR so we can just get all these bottles published now.